scanner.py

```diff
@@ -22,7 +28,7 @@
 # scanning the port only in range of (1, 8888)
 
 try:
-    for port in range(1,8888):  
+    for port in range(low,high):  
```

Comment:
Linting issue, follow the same rule.
@ishika1727

Reply:
Done. Please check. Thank you.